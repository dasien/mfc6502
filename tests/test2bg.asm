; Perform simple addition and test transfers from accumulator to x & y registers.

LDA #01
TAX
LDA #02
TAY
ADC #01
