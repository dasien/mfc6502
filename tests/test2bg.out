A9 01
AA 
A9 02
A8 
69 01
